apps/server/.env.example

```
@@ -1,6 +1,8 @@
 BETTER_AUTH_GOOGLE_CLIENT_ID=xxxxx
 STRIPE_WEBHOOK_SECRET=xx
 STRIPE_SECRET_KEY=xxx
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The STRIPE_SECRET_KEY key should go before the STRIPE_WEBHOOK_SECRET key

(UnorderedKey)